src/software/simulated_tests/terminating_validation_functions/robot_in_center_circle_validation.cpp

```diff
@@ -20,6 +20,7 @@ void robotInCenterCircle(RobotId robot_id, std::shared_ptr<World> world_ptr,
 
     while (!robot_in_center_circle(world_ptr))
     {
-        yield();
+        yield("Robot with ID " + std::to_string(robot_id) +
+              " did not enter the center circle");
```

Comment:
Maybe change this to "has not entered the center circle", since this is a continuous check rather than a one-off where the robot did not enter the center circle

Reply:
fixed!